app/models/ncr/work_order.rb

```diff
@@ -102,8 +102,17 @@ def setup_approvals_and_observers(approving_official_email)
         # skip state machine
         self.proposal.update(status: 'approved')
       else
-        approvers = emails.map{|e| User.for_email(e)}
-        self.proposal.approvers = approvers
+        individuals = emails.map do |email|
+          user = User.for_email(email)
+          # Reuse existing approvals, if present
+          if existing = self.proposal.existing_approval_for(user)
+            existing
+          else
+            Approvals::Individual.new(user: user)
+          end
+        end
+
+        self.proposal.root_approval = Approvals::Serial.new(child_approvals: individuals)
```

Comment:
Do we need to worry about a `root_approval` already existing here, or can we safely assume this will only be called on the initial setup?